### ISSUE TITLE
WAI-ARIA spec does not mention the default value of aria-checked for menuitemradio

### ISSUE THREAD

**Issue by ianthedev** [mdn/content#26092]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/menuitemradio_role

### What specific section or headline is this issue about?

Description

### What information was incorrect, unhelpful, or incomplete?

> If missing, the value defaults to `false`.

The WAI-ARIA spec for [menuitemradio](https://w3c.github.io/aria/#menuitemradio) and [aria-checked](https://w3c.github.io/aria/#aria-checked) do not mention that.

### What did you expect to see?

I expected to see the same statement in the WAI-ARIA spec for [menuitemradio](https://w3c.github.io/aria/#menuitemradio) or [aria-checked](https://w3c.github.io/aria/#aria-checked).

### Do you have any supporting links, references, or citations?

WAI-ARIA [menuitemradio](https://w3c.github.io/aria/#menuitemradio) and [aria-checked](https://w3c.github.io/aria/#aria-checked).

### Do you have anything more you want to share?

The [ARIA: menuitemcheckbox role](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/menuitemcheckbox_role) page has the same issue.

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/accessibility/aria/roles/menuitemradio_role`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/menuitemradio_role
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/accessibility/aria/roles/menuitemradio_role/index.md
* Last commit: https://github.com/mdn/content/commit/5555af07f3ffd3918667d6d614a0f12aaf664ce8
* Document last modified: 2023-03-01T05:15:58.000Z

</details>
